### PR TITLE
Removed some sentences from child name

### PR DESCRIPTION
**Pull request by leachtucker**
# Description
Long sentences were included in childrens' names

Fixes #

- What work was done?
Removed some sentences from seed data and placed them into comments.
- Why was this work done?
The long names were causing the front-end client to look unsightly.
- What feature / user story is it for?
_NA_
- Any relevant links or images / screenshots

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)

## Change Status

- [X] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [X] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] There are no merge conflicts

### CODE REVIEW

**Comment by leachtucker** [comment]
The build tests will not complete successfully until Lambda fixes the Repo's secrets/env variables on the Settings page...

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -32,8 +32,6 @@ jobs:
       # Run the NPM install command before proceeding
       - run: npm install
 
-      - run: npm run lint
```
Comment (leachtucker) [inline .github/workflows/ci.yml]: Consistently failing and unnecessary to run on build

**Comment by leachtucker** [comment]
[As a user, I want a list of three-four children associated with my account.](https://trello.com/c/sMqN07jc/49-as-a-user-i-want-a-list-of-three-four-children-associated-with-my-account)

**Review by Robert-Lark — DISMISSED** [review]
Looks good

File: .github/workflows/ci.yml (hunk 1 above)
Comment (charlene-johnson) [reply .github/workflows/ci.yml]: Everything looks good

**Review by charlene-johnson — DISMISSED** [review]

**Pull request closed without merging**